tests/postgres_tests/test_bulk_update.py

```diff
@@ -16,6 +18,7 @@
     pass  # psycopg2 isn't installed.
 
 
+@modify_settings(INSTALLED_APPS={"append": "django.contrib.postgres"})
```

Comment:
Can you reproduce any failure without `modify_settings()`? This test doesn't need anything from `PostgresConfig.ready()` :game_die: :thinking:

Reply:
Yes -- see gist https://gist.github.com/smithdc1/e2a1dbae04d2e06391bbde129e1e21ad

Reply:
OK, so `PostgresConfig.ready()` enforces calling `register_type_handlers` that we need for `hstore`. It should also be called when we create a new extension, I'm not sure why it's not called with `spawn` :thinking: .